Makefile

```diff
@@ -97,7 +97,7 @@ KUBEBUILDER_ASSETS ?= $(ENVTEST_ASSETS_DIR)/k8s/$(ENVTEST_K8S_VERSION)-$(TARGET_
 # Ginkgo configuration
 GINKGO_NODES ?= 12
 GINKGO_EDITOR_INTEGRATION ?= true
-GINKGO_OPTS = -vv --randomize-all --output-interceptor-mode=none --trace --timeout 90m --flake-attempts=1 --race --nodes=$(GINKGO_NODES) --cover --coverpkg=github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/...
+GINKGO_OPTS = -vv --randomize-all --output-interceptor-mode=none --trace --timeout 90m --flake-attempts=1 --race --nodes=$(GINKGO_NODES) --cover --coverpkg=github.com/mongodb/mongodb-atlas-kubernetes/v2/...
```

Comment:
These Makefile fixes, while welcome, do not seem related to the title of the PR, even though they might be required. For this kind of things I'd rather either have a separate PR or put the fix on its own separated commit within the same PR.